src/components/AddTask/AddTask.js

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+import Modal from '@mui/material/Modal';
+import { useDispatch, useSelector } from 'react-redux';
+import { addTaskToProject } from '../../reducers/user';
+import { v4 as uuidv4 } from 'uuid';
+
+const style = {
+  position: 'absolute',
+  top: '50%',
+  left: '50%',
+  transform: 'translate(-50%, -50%)',
+  width: 400,
+  bgcolor: 'background.paper',
+  border: '2px solid #000',
+  boxShadow: 24,
+  p: 4,
+};
+
+function AddTask({ taskStatus }) {
+  const dispatch = useDispatch();
+  const [open, setOpen] = React.useState(false);
+  const [newTaskTitle, setNewTaskTitle] = React.useState('');
+  const [newTaskDescription, setNewTaskDescription] = React.useState('');
+
+  const handleOpen = () => setOpen(true);
+  const handleClose = () => {
+    setOpen(false);
+    setNewTaskTitle('');
+    setNewTaskDescription('');
+  }
+
+  const handleAddTask = () => {
+    console.log("Adding task:", newTaskTitle, newTaskDescription, taskStatus)
+    dispatch(addTaskToProject({
+        task_id: uuidv4(), // random uuid
```

Comment:
For the random uuid, maybe there's a different library that the BE team is using